package.json

```diff
@@ -46,8 +46,7 @@
     "remark-mdx-searchable": "^0.1.3",
     "theme-ui": "^0.7.5",
     "unified": "^10.1.2",
-    "unist-util-visit": "^4.1.0",
```

Comment:
are there other dependencies that should be removed now that we are removing the amplify backend?  Should we remove the SubmissionLambda script?

Reply:
Thanks Jacob, yes I have that in my second PR to remove our Amplify code. I don't think we can remove all of the Amplify packages though because `@aws-amplify/ui-react` depends on them